iohk/CardanoCSL.hs

```diff
@@ -125,16 +125,6 @@ dumpLogs o c withProf nodes = do
              , defLogs
              ]
 
-generateIPDHTMappings :: Options -> NixopsConfig -> IO Text
```

Comment:
This shouldn't be dropped since it's useful for development and testing a feature with a custom installer.